src/auditlog/compat.py

```diff
@@ -0,0 +1,15 @@
+import django
+
+def is_authenticated(user):
+    """
+    Return whether or not a User is authenticated.
+
+    Function provides compatibility following deprecation of method call to
+    is_authenticated() in Django 2.0.
+
+    """
+
+    if django.VERSION < (1, 10):
```

Comment:
This works perfectly, however, checking if `user.is_authenticated` is callable might work too.

No need to change this (obviously) unless you'd like to, kind of a personal preference I guess.

Reply:
No problem to change it (tested change and all OK). 

Thought behind the original version was to make it clear why the code existed — that way when you drop support for <1.10 in the future it's more obvious what code can be removed from `compat.py`.

Reply:
personally id prefer the `if django.VERSION < (1, 10)` code for the reason stated, it lets you know when you can drop it in the compat, whereas looking at the new code I have no idea why it was added or what versions of django it makes compatible without added comments.